cylc/flow/etc/syntax/cylc.nanorc

```diff
@@ -0,0 +1,30 @@
+# How to use this file:
+# cylc get-resources cylc.nanorc ~/.config/nano/cylc.nanorc
+# Add the following to ~/.nanorc
+# include ~/.config/nano/cylc.nanorc
+
+# Supports `.cylc` files
+syntax "Cylc" "\.cylc$"
+
+## Multiline
+color yellow start="\"\"\"" end="\"\"\""
+color yellow start="'''" end="'''"
+color yellow start="\[" end="\]"
+
+## Values
+color yellow "=(.*)$"
+color green "=[^>]"
```

Comment:
What's this one about?

Reply:
`=` signs that aren't `=>` so we don't include graph deps.